src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -39,7 +39,7 @@ public class MicrosoftIdentityOptions : OpenIdConnectOptions
         /// If you don't specify the redirect URI, the redirect URI will be computed from the URL on which the app is
         /// deployed and the CallbackPath.
         /// </summary>
-        public string RedirectUri { get; set; }
+        public string? RedirectUri { get; set; }
```

Comment:
> public string? RedirectUri { get; set; } [](start = 7, length = 41)

need to update w/master, these have been removed.

Reply:
Rebased.